docs/DeveloperGuide.md

```diff
@@ -283,7 +283,7 @@ of a `StatisticsCommand`:
 
 ### Design Consideration:
 
-#### Aspect 1: How statistics feature executes
```

Comment:
Would it be good to keep this? Because there might be multiple aspects to a feature